src/index.test.ts

```diff
@@ -1,10 +1,16 @@
-import { deepEqual } from "node:assert/strict";
```

Comment:
If the formatter is removing the semicolons, make sure to configure your editor to use the default Prettier settings, or there will be trouble in the course with different people using different formatters.